BasicHistoryApi/app/src/main/res/layout/activity_main.xml

```diff
@@ -1,11 +1,11 @@
 <!--
-  Copyright 2014 Google, Inc.
+  Copyright 2019 Google LLC
```

Comment:
Again, you don't have to do this. You can leave changes this time if you want, I just wouldn't worry about it moving forward.